graph/src/data/subgraph/schema.rs

```diff
@@ -1,102 +1,89 @@
 //! See `core/src/subgraph/subgraphs.graphql` for corresponding graphql schema.
 
 use super::SubgraphId;
-use components::store::{EntityKey, EntityOperation, Store};
-use data::store::Value;
+use components::store::{EntityKey, EntityOperation};
+use data::store::{Entity, Value};
 use failure::Error;
 use std::collections::HashMap;
 
 /// ID of the subgraph of subgraphs.
-pub const SUBGRAPHS_ID: &str = "subgraphs";
+lazy_static! {
+    pub static ref SUBGRAPHS_ID: SubgraphId = SubgraphId::new("subgraphs").unwrap();
+}
 
 /// Type name of the root entity in the subgraph of subgraphs.
 pub const SUBGRAPH_ENTITY_TYPENAME: &str = "Subgraph";
 
-const EVENT_SOURCE: &str = "subgraph-added";
-
 #[derive(Debug)]
 pub struct SubgraphEntity {
     id: SubgraphId,
-    manifest: SubgraphManifest,
+    manifest: SubgraphManifestEntity,
     created_at: u64,
 }
 
 impl SubgraphEntity {
     pub fn new(source_manifest: &super::SubgraphManifest, created_at: u64) -> Self {
         Self {
             id: source_manifest.id.clone(),
-            manifest: SubgraphManifest::from(source_manifest),
+            manifest: SubgraphManifestEntity::from(source_manifest),
             created_at,
         }
     }
 
-    pub fn write_to_store(self, store: &impl Store) -> Result<(), Error> {
+    pub fn write_ops(self) -> Vec<EntityOperation> {
```

Comment:
I'm ok calling variables `something_ops` but could we spell this one out and make it `write_operations`?

Reply:
Done!